### ISSUE TITLE
Can I set The timeout time so tokens never expire?

### ISSUE THREAD

**Issue by ShayanJa** [appleboy/gin-jwt#81]
I want the user to be logged out after a week of not using the site. How can I do this?

**Comment by zhaowangdaren** [comment]
Refresh the token at the end of the week.

**Comment by gonboy** [comment]
even you refresh the token,  the old token is also can be use.
so maybe you can create a blacklist to block the old token

**Comment by DevAlone** [comment]
You can pass something like this to Timeout:

`Timeout:    time.Duration(24*365) * time.Hour, // one year`

I'm not sure if it's good solution, but it works :)

**Comment by viseth** [comment]
@DevAlone yes but this will set the timeout for one year only, what happens after one year...

@appleboy : thanks for this amazing work. For setting token that never expires, do you provide something like "Timeout: 0,"?

**Comment by bery** [comment]
@viseth why don't you try to set expiration to 100 years?
